homeassistant/components/egps/config_flow.py

```diff
@@ -0,0 +1,56 @@
+"""ConfigFlow for EGPS devices."""
+import logging
+from typing import Any
+
+from pyegps import get_device, search_for_devices
+from pyegps.exceptions import MissingLibrary, UsbError
+import voluptuous as vol
+
+from homeassistant import config_entries
+
+from .const import CONF_DEVICE_API_ID, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
any reason why you're not using the one from const here?

Reply:
I should do that, you are right